tests/integration/targets/alternatives/tasks/tests_state.yml

```
@@ -0,0 +1,71 @@
+# Add a few dummy alternatives with state = present and make sure that the
+# group is in 'auto' mode and the highest priority alternative is selected.
+- name: Add some dummy alternatives with state = present
+  alternatives:
+    name: dummy
+    path: "/usr/bin/dummy{{ item.n }}"
+    link: /usr/bin/dummy
+    priority: "{{ item.priority }}"
+    state: present
+  loop:
+    - { n: 1, priority: 50 }
+    - { n: 2, priority: 70 }
+    - { n: 3, priority: 25 }
+
+- name: Ensure that the link group is in auto mode
```

How about printing some more info:
```suggestion
- name: Get contents of file
  command: cat {{ alternatives_dir }}/dummy

- name: Ensure that the link group is in auto mode
```
While running the tests on a RHEL 8.4 remote I was able to see:
```
/var/lib/alternatives$ cat dummy 
manual
/usr/bin/dummy

/usr/bin/dummy3
25
/usr/bin/dummy4
64
/usr/bin/dummy1
50
/usr/bin/dummy2
70
```
Having a `cat` directly before the command below would be more conclusive, though.